_config.yml

```diff
@@ -46,13 +46,7 @@ defaults:
   scope:
     path: "" # an empty string here means all files in the project
   values:
-    layout:
-      - home
-      - page
-      - "project-list"
-      - project
-      - "team-list"
-      - team
+    layout: page
```

Comment:
Assigning a list to `layout` doesn't seem particularly useful, since it just makes jekyll complain that the list is an invalid layout when a document's front-matter doesn't contain a `layout` key.